example/samples/install_ubuntu_payload_iso_minimal.json

```diff
@@ -0,0 +1,12 @@
+{
+    "options": {
+        "defaults": {
+            "repo": "http://{{ api.server }}/ubuntu",
```

Comment:
{{api.server}} has already contained `http://`, so you need remove it. see https://github.com/RackHD/on-tasks/blob/master/lib/task.js#L115